packages/compartment-mapper/src/archive.js

```diff
@@ -105,6 +105,10 @@ const translateCompartmentMap = (compartments, sources, renames) => {
           modules[name] = {
             exit,
           };
+        } else if (deferredError !== undefined) {
+          modules[name] = {
+            deferredError,
+          };
```

Comment:
I’m okay with `error` for the name.

Reply:
I might change it, but at least for now it's been much easier to navigate the codebase with a more unique name for it. I see value in that.

Reply:
I propose keeping it. Much easier to track across multiple files than just `error`

Reply:
I support this choice. Thanks for considering.